lib/v2/screens/explore_screens/vote_screens/proposal_details/proposal_details_screen.dart

```diff
@@ -0,0 +1,254 @@
+                                ),
+                                TextSpan(
+                                  text: proposal.url,
+                                  style:
+                                      Theme.of(context).textTheme.subtitle3OpacityEmphasis.copyWith(color: Colors.blue),
+                                  recognizer: TapGestureRecognizer()
+                                    ..onTap = () async {
+                                      if (await launcher.canLaunch(proposal.url)) {
+                                        await launcher.launch(proposal.url);
+                                      } else {
+                                        // todo listener snack
+                                        // print("Couldn't open this url".i18n);
+                                      }
+                                    },
+                                ),
+                              ],
+                            ),
+                          ),
+                          const SizedBox(height: 8.0),
+                          Text('Description'.i18n, style: Theme.of(context).textTheme.headline7),
+                          const SizedBox(height: 8.0),
+                          SelectableText(
+                            proposal.description,
+                            style: Theme.of(context).textTheme.subtitle3OpacityEmphasis,
+                          ),
+                        ],
+                      ),
+                    ),
+                    const DividerJungle(),
+                    Padding(
+                      padding: const EdgeInsets.all(16.0),
+                      child: Column(
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: [
+                          Text('Precast your Vote'.i18n, style: Theme.of(context).textTheme.headline7),
+                          const SizedBox(height: 25.0),
+                          Padding(
+                            padding: const EdgeInsets.symmetric(horizontal: 24.0),
+                            child: Row(
+                              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                              children: [
+                                Column(
+                                  children: [
+                                    InkWell(
+                                      borderRadius: BorderRadius.circular(32.5),
+                                      onTap: () {},
+                                      child: const CustomPaint(
+                                        size: Size(65, 65),
+                                        painter: VotesUpArrow(
+                                          circleColor: AppColors.lightGreen6,
+                                          arrowColor: AppColors.white,
+                                        ),
+                                      ),
+                                    ),
```

Comment:
InkWeell..
It seems not to work with the Painters, the splash effect is not seen, I was investigating but I did not find anything easy to use to solve it.
If the splash effect is something really needed then here (much to my regret) it will have to be SVGs.